frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseOverrideForkPointAction.java

```diff
@@ -63,8 +63,8 @@ public void onUpdate(AnActionEvent anActionEvent) {
   public void actionPerformed(AnActionEvent anActionEvent) {
     var project = getProject(anActionEvent);
     var selectedVcsRepository = getSelectedGitRepository(anActionEvent).getOrNull();
-    var branchUnderAction = getNameOfBranchUnderAction(anActionEvent);
-    var branch = branchUnderAction.flatMap(pn -> getGitMacheteBranchByName(anActionEvent, pn)).getOrNull();
+    var branchUnderAction = getNameOfBranchUnderActionWithLogging(anActionEvent);
+    var branch = branchUnderAction.flatMap(pn -> getGitMacheteBranchByNameWithLoggingOnEmpty(anActionEvent, pn)).getOrNull();
```

Comment:
inconsistency alert :exclamation: 

`<something>WithLogging`
VS
`<something>WithLoggingOnEmpty`

I think that we could you the first one since it is shorter